tests/lsf/mocked_jobs/mock_bjobs

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import datetime
+import json
+import os.path
+import sys
+
+timestamp = str(datetime.datetime.now())
+
+with open("job_paths", "r") as job_paths:
```

Comment:
It would be neat if we could get the indentation level of this code down a bit. A good start would be to parse `job_paths` and close it again immediately :)